modelon/impact/client/entities/experiment.py

```diff
@@ -327,12 +327,16 @@ def get_cases_with_label(self, case_label: str) -> List[Case]:
         """
         return [case for case in self.get_cases() if case.meta.label == case_label]
 
-    def get_trajectories(self, variables: List[str]) -> Dict[str, Any]:
+    def get_trajectories(
+        self, variables: List[str], last_point_only: bool = False
+    ) -> Dict[str, Any]:
```

Comment:
I think from user facing API perspective it's preferable to have a separate method and a different datatype. There's no reason to return "trajectory" objects when we only have a single number. Have you considered returning numpy array or dataframe instead?